jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -246,6 +246,25 @@ ecma_op_general_object_get_property (ecma_object_t *obj_p, /**< the object */
 } /* ecma_op_general_object_get_property */
 
 /**
+ * Checks whether the property name is "length".
+ *
+ * @return true if the property name is length
+ *         false otherwise
+ */
+static bool
+ecma_op_general_object_property_name_is_length (ecma_string_t *property_name_p) /**< property name */
```

Comment:
IMHO, until the JERRY_ASSERT is not defined as empty the compiler won't inline this because it is called twice. We should use `inline` and `__attr_always_inline___` to force it.